### PR TITLE
add profile for bastian

### PR DESCRIPTION
**Pull request by bzapf**
suggestions for improvement of my profile text, if necessary, are very welcome

### CODE REVIEW

File: content/about/employees/bastian/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Bastian holds a Ph.D. in computational mathematics from the University of Oslo. 
+In his thesis he used mathematical modeling to investigate how molecules are transported in the human brain.
+His expertise lies in numerical simulations, nonlinear optimization and deep learning.
+Bastian is a systematic person with interest in computational modeling and working with imaging data.
+<!--more-->
+
+From his academic work, Bastian has gained substantial experience in combining large amounts of clinical imaging data with computational models.
+He is used to performing resource--demanding computations on supercomputers, including numerical simulations and training deep neural networks. 
+Being a pragmatic programmer eager to learn new techniques, he used shape optimization methods in a data--driven approach to simplify the intricate workflow of his simulations.
```
Comment (cebasfu93) [inline content/about/employees/bastian/index.md:19]:
```suggestion
Being a pragmatic programmer eager to learn new techniques, he used shape optimization methods in a data-driven approach to simplify the intricate workflow of his simulations.
```

**Review by cebasfu93 — APPROVED** [review]
lgtm. I spotted one typo, though :)

**Review by cebasfu93 — APPROVED** [review]

**Comment by alexanfl** [comment]
@bzapf can this be merged now? :)

**Review by skavhaug — COMMENTED** [review]
LGTM!

**Pull request merged**